src/cmds/scripts/pbs_postinstall.in

```diff
@@ -806,7 +821,7 @@ create_home() {
 echo "*** PBS Installation Summary"
 echo "***"
 echo "*** Postinstall script called as follows:"
-echo "*** $0 $*"
+printf "*** $0 "; printf "%q " "$@"; printf "\n"
```

Comment:
This fixes an issue that was introduced when "sameconf" was added to address PP-441. Because an empty parameter is supplied by pbs_habitat when it calls pbs_postinstall it was not showing up in the output. This doesn't change the output so much as corrects it to show the empty parameter. I don't think this needs to be mentioned in the external design.